package.json

```diff
@@ -2,8 +2,11 @@
   "name": "retry-axios",
   "version": "0.0.0",
   "description": "Retry HTTP requests with Axios.",
-  "main": "./build/src/index.js",
-  "types": "./build/src/index.d.ts",
+  "source": "src/index.ts",
```

Comment:
What is `source`?

Reply:
It tells microbundle where to find the source file to bundle.

> Unless overridden via the command line, microbundle uses the `source` property in your `package.json` to locate the input file, and the `main` property for the output.
https://github.com/developit/microbundle#microbundle--microbundle-build

Reply:
Ok, so this is only consumed at build time?  Not by downstream consumers?  I ask because the `src` directory is *not* included in the npm module.

Reply:
Correct, this runs during the NPM's prepare phase.

Yes, I killed `build/src`.
Okay, so is `src` needed by downstream consumers?
With this change I replaced `build/src` with `dist` and kept `build/*` for running tests.

So far consumers received this tarball via npm:
https://registry.npmjs.org/retry-axios/-/retry-axios-2.1.3.tgz

Reply:
@JustinBeckwith If you prefer it not to be declarative as a source field.
There is also a cli option available for `--entry` (`-i`)
https://github.com/developit/microbundle#all-cli-options

Reply:
Nah `source` is fine, I just wanted to make sure none of the tools here required it at runtime (as opposed to build time).